README.md

```diff
@@ -21,6 +35,3 @@ Fork this repository, clone it to your local machine and run the following comma
 > bin/rails server
```

Comment:
Would be helpful if we  mention the command to migrate the tables in the test environment
bin/rails db:migrate RAILS_ENV=test

Reply:
As the project currently stands in master I don't think this command is necessary. If this comes up in a future time because of a code change then yes agree but right now I don't think this is necessary.